builder/builder.go

```diff
@@ -1312,6 +1311,39 @@ func (b *Builder) createMixBundleDir() error {
 	return nil
 }
 
+type dnfConf struct {
+	UpstreamURL, RepoDir string
```

Comment:
I like doing this in function signatures... I feel conflicted about this in a struct. :man_shrugging: looks weird :)